cmd/zpool/zpool_main.c

```diff
@@ -6467,6 +6467,15 @@ status_callback(zpool_handle_t *zhp, void *data)
 		    "to be recovered.\n"));
 		break;
 
+	case ZPOOL_STATUS_IO_FAILURE_MMP:
+		(void) printf(gettext("status: The pool is suspended because "
+		    "multihost writes failed or were delayed, and another "
+		    "system could import the pool undetected.\n"));
+		(void) printf(gettext("action: Make sure the pool's devices "
+		    "are connected, then reboot your system and import the "
+		    "pool.\n"));
```

Comment:
Please mind the 80 char limit in the output, both of these lines need to be manually wrapped with `\n\t\`.

Reply:
Fixed.